extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/caffeine/metrics/MetricsInitializer.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.cache.runtime.caffeine.metrics;
+
+import com.github.benmanes.caffeine.cache.AsyncCache;
+
+public interface MetricsInitializer {
+
+    boolean metricsEnabled();
+
+    void recordMetrics(AsyncCache<Object, Object> cache, String cacheName, String[] tags);
```

Comment:
I think you really should consider using MeterFilter (alone) to manage custom tags.. as that can apply to any/all metrics (based on the rule they apply), rather than only to cache..

Reply:
I discovered `MeterFilter` and tested today. It is indeed much better than having cache tags declared in `applications.properties`. Thanks for the suggestion!